sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveQlSuite.scala

```diff
@@ -195,4 +195,11 @@ class HiveQlSuite extends SparkFunSuite with BeforeAndAfterAll {
 
     assert(plan.children.head.asInstanceOf[Generate].generator.isInstanceOf[JsonTuple])
   }
+
+  test("parse union distinct/except/intersect") {
+    HiveQl.parseSql("select a, b from t1 union all t2 union all t3")
+    HiveQl.parseSql("select a, b from t1 union distinct t2 union distinct t3")
```

Comment:
Actually this test case shouldn't be here? It should just go into the normal sql test case now.